projects/packages/jetpack-mu-wpcom/src/features/launchpad/class-launchpad-task-lists.php

```diff
@@ -445,6 +450,38 @@ private function load_subtitle( $task ) {
 		return '';
 	}
 
+	/**
+	 * Loads the extended content for a task.
+	 *
+	 * @param Task $task A task definition.
+	 * @return array|null The extended content for the task.
+	 */
+	private function load_extended_content( $task ) {
+		if ( ! isset( $task['extended_content'] ) ) {
+			return null;
+		}
+
+		$extended_content_context = array();
+
+		foreach ( $task['extended_content'] as $context => $extended_content ) {
```

Comment:
I _really_ dislike the variable name here, because we're using it as a callback and not as the content itself. I'd strongly prefer something like `$extended_content_callback`.
```suggestion
		foreach ( $task['extended_content'] as $context => $extended_content_callback ) {
```